npm-scripts.mjs

```diff
@@ -315,6 +322,20 @@ function buildWorker() {
 	executeCmd(`"${PYTHON}" -m invoke -r worker mediasoup-worker`);
 }
 
+function buildWorkerLib() {
+	logInfo('buildWorkerLib()');
+
+	installInvoke();
+
+	executeCmd(`"${PYTHON}" -m invoke -r worker libmediasoup-worker`);
+
+	const buildType = process.env.MEDIASOUP_BUILDTYPE || 'Release';
+
+	executeCmd(
+		`cd node/workerChannel && GYP_DEFINES="mediasoup_build_type=${buildType}" npm i`
```

Comment:
Can we use `npm ci` instead please? And copy `.npmrc` from root folder to `node/workerChannel`.

Reply:
Also, the prebuilt stuff is missing totally. We will need to upload the library now, not the binary.